languages/js/src/Polar.ts

```diff
@@ -185,8 +185,13 @@ export class Polar {
   /**
    * Query for a Polar predicate or string.
    */
-  query(q: Predicate | string, bindings?: Map<string, unknown>): QueryResult {
+  query(
+    q: Predicate | string,
+    bindings?: Map<string, unknown>,
+    acceptExpression?: boolean
```

Comment:
What do you think about an `options` object as the final parameter here? So the call looks like

```ts
query('f(x)', {
  bindings: myBindings,
  acceptExpression: true
})
```

Reply:
I think that makes a lot of sense. I have refactored the code with this. Thanks for your suggestions!